pyscal/factory.py

```diff
@@ -400,7 +400,9 @@ def create_gas_oil(params=None, fast=False):
         return gasoil
 
     @staticmethod
-    def create_water_oil_gas(params=None, fast=False):
+    def create_water_oil_gas(
+        params: Dict[str, float] = None, fast: bool = False
```

Comment:
Does `mypy` accept cases like this with optional parameters with default value `None`? 🤔 In my experience it requires something like this instead:
```suggestion
        params: Optional[Dict[str, float]] = None, fast: bool = False
```